src/store/modules/farmer/state.ts

```diff
@@ -0,0 +1,10 @@
+import { FarmerState, Farmer } from './types';
+import UTCDate from '../../../utils/UTCDate';
+
+const initialState: FarmerState = {
+  containsLocal: false,
+  farmers: [] as Farmer[],
```

Comment:
How is `[] as Farmer[]` different than just `Farmer[]`?

Reply:
`[] as Farmer[]` is a literal array `[]` as a type farmer. Without casting the empty array to a type, it assumes type `any[]`. Something similar w/ a variable declaration would be `let a: number[] = []`